samples/sensor/amg88xx/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _amg88xx:
+
+AMG88XX Infrared Array Sensor
+#############################
+
+Overview
+********
+
+Sample application that periodically reads 8x8 temperature array from
+the AMG88XX sensor. Sample can also be configured to be triggered
+when the upper or lower threshold has been reached.
```

Comment:
I'm confused a bit here.  Is the upper/lower threshold an application-supplied value that will cause a trigger when it get's reached?  If so, where does the application specify this value?